akka-actor/src/main/resources/reference.conf

```diff
@@ -547,14 +547,16 @@ akka {
       # FQCN of the MailboxType. The Class of the FQCN must have a public
       # constructor with
       # (akka.actor.ActorSystem.Settings, com.typesafe.config.Config) parameters.
-      mailbox-type = "akka.dispatch.UnboundedMailbox"
+      mailbox-type = "akka.dispatch.VirtuallyUnboundedJCToolsMailbox"
```

Comment:
That sounds to me too much like adding dead code. The improvements are great when they are enabled by default, otherwise, it's just too small an improvement that we should care to maintain a separate copy of JCTools etc.

Reply:
Perhaps we should just write a blog post about it? The amount of code fits nicely within a blog post.

Reply:
You mean how to implement the mailbox in your own code?

Reply:
yes